private/src/scripts/editor/blocks/header/DisplayComponent.jsx

```diff
@@ -76,7 +76,11 @@ class DisplayComponent extends Component {
     wp.apiRequest({
       path: `/wp/v2/media/${featuredVideoId}`,
     }).then((resp) => {
-      this.setState({ videoUrl: resp.source_url });
+      this.setState({
+        videoUrl: resp.source_url,
+        videoCaption: resp.caption.rendered.replace(/<[^>]+>/g, ''),
+        videoDescription: resp.description.rendered.replace(/<[^>]+>/g, ''),
```

Comment:
stripping html in this way isn't robust. you could instead query the edit context, which provides the raw text without markup (below i've also limited the fields to those that are being used, to reduce the payload):

```
const { addQueryArgs } = wp.url;

const path = addQueryArgs(`/wp/v2/media/${featuredVideoID}`, {
  context: 'edit',
  _fields: 'id,source_url,caption,description',
});

wp.apiRequest({ path }).then((resp) => {
  this.setState({
    videoUrl: resp.source_url,
    videoCaption: resp.caption.raw,
    videoDescription: resp.description.raw,
  });
});
```

Reply:
resolved